src/tests/components/NewArticleForm.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import NewArticleForm from '../../components/Articles/NewArticleForm';
+
+describe('NewArticleForm component', () => {
+  const mockStore = configureMockStore();
+  let wrapper;
+  const handleSubmit = jest.fn();
+  const resetForm = jest.fn();
+  const handleChange = jest.fn();
+  const handleEditorChange = jest.fn();
+  const title = '';
+  const description = '';
+  const body = '';
+  const loading = false;
+
+  beforeEach(() => {
+    mockStore({});
+    wrapper = shallow(<NewArticleForm
+      handleSubmit={handleSubmit}
+      resetForm={resetForm}
+      handleChange={handleChange}
+      handleEditorChange={handleEditorChange}
+      body={body}
+      title={title}
+      description={description}
+      loading={loading}
+    />);
+  });
+
```

Comment:
This line too should be removed and all the other redundant lines.

Reply:
@kakaemma Redundant line 31 has been removed. Please note that there are some other blank lines left for easy readability of code.